glue-vispy-viewers/bld.bat

```diff
@@ -0,0 +1,9 @@
+%PYTHON% setup.py install
+if errorlevel 1 exit 1
+
+del %SCRIPTS%\easy_install*
+
+if %PY3K%==1 (
+    rd /s /q %SP_DIR%\__pycache__
+    rd /s /q %SP_DIR%\PyQt5
+)
```

Comment:
I'm not sure whether all these lines are needed, but I was copying from the glueviz recipe